sample/build.gradle.kts

```diff
@@ -8,20 +7,24 @@ plugins {
 }
 
 dependencies {
-    implementation(Kotlin.stdlib)
+    // fuel related libraries
```

Comment:
these should be implementation because api should be only in the libraries.

Reply:
it is a sample app, so ... 🤷‍♂️ ww

Reply:
Addressed!